<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendsWindow</class>
 <widget class="QWidget" name="FriendsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_rcvmsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sndmsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_send_msg">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_add_friend">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_del_friend">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_user">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>260</width>
     <height>470</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>470</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>470</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_allonline">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>在线列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_onlinefriend">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_search">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
